app/templates/whats-new.jade

```diff
@@ -1,4 +1,12 @@
-li(ng-if="status.didUpgradeToHd" ng-class="::{flash: feat.date > lastViewedWhatsNew}" ng-repeat="feat in feats | orderBy:'-date' track by $index")
+li(
+  ng-if="status.didUpgradeToHd"
+  ng-repeat="feat in feats | orderBy:'-date' track by $index"
+  ng-hide="feat.title === 'BUY_BITCOIN' && !showBuyWhatsNew"
```

Comment:
Can you modify the list in the controller instead of the view?

How does the unread count behave?